src/arkhamdb/DeckImporterMain.ttslua

```diff
@@ -422,27 +166,11 @@ function loadCards(slots, investigatorId, customizations, playerColor, commandMa
     for cardId, remainingCount in pairs(slots) do
       if remainingCount > 0 then
         hadError = true
-        local request = Request.start({
-          configuration.api_uri,
-          configuration.cards,
-          cardId
-        },
-          function(result)
-            local adbCardInfo = JSON.decode(fixUtf16String(result.text))
-            local cardName = adbCardInfo.real_name
-            if (cardName ~= nil) then
-              if (adbCardInfo.xp ~= nil and adbCardInfo.xp > 0) then
-                cardName = cardName .. " (" .. adbCardInfo.xp .. ")"
-              end
-              debugPrint("Card not found: " .. cardName .. ", ArkhamDB ID " .. cardId, Priority.ERROR, playerColor)
-            else
-              debugPrint("Card not found in ArkhamDB, ID " .. cardId, Priority.ERROR, playerColor)
-            end
-          end)
+        arkhamDb.logCardNotFound(cardId, playerColor)
```

Comment:
Instead of using a separate `hadError` variable, the coroutine could end with `return 0` at this point

Reply:
Ah, I missed the loop because it wasn't changed 😅
At the end of the coroutine the returned value should still probably not be `1` if there was an error.